daemon/container_operations.go

```diff
@@ -455,6 +482,28 @@ func (daemon *Daemon) allocateNetwork(ctx context.Context, cfg *config.Config, c
 		}
```

Comment:
Not new code, and can't comment on the actual line above, but wondering if we should log if we fail to delete the sandbox.

Reply:
Ok, done.